site/dev/common.sh

```diff
@@ -85,6 +85,26 @@ assert_not_empty () {
   fi
 }
 
+# Creates a 'nightly' version of the documentation that points to the current versioned docs
+# located at the root-level `/docs` directory.
+create_nightly () {
+  echo " --> create nightly"
+
+  # Remove any existing 'nightly' directory and recreate it
+  rm -rf docs/docs/nightly/
+  mkdir docs/docs/nightly/
+
+  # Create symbolic links and copy configuration files for the 'nightly' documentation
+  ln -s "../../../../docs/docs/" docs/docs/nightly/docs
+  cp "../docs/mkdocs.yml" docs/docs/nightly/
```

Comment:
can we make this a symlink as well? Otherwise you won't see changes made to the navigation